spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -51,13 +57,30 @@ definitions:
             units: cycles / 256
             desc: Carrier phase fractional part
 
+ - Doppler:
+    short_desc: GNSS doppler measurement.
+    desc: |
+      Doppler measurement in Hz represented as a 24-bit
+      fixed point number with Q16.8 layout, i.e. 16-bits of whole
+      doppler and 8-bits of fractional doppler.  This doppler is defined
+      as positive for approaching satellites.
+    fields:
+        - i:
+            type: s16
```

Comment:
This won't work in orbit.  Something to keep in mind, though not a deal breaker.